GUI

```diff
@@ -211,13 +213,25 @@ public class Gui extends JPanel {
 	Object[][] 	data;
 //build a column include the select column for each activity		
 	String[][] data_m1	= new String[data_length][activity_length];
+    
+
 	String[][] data_m2	= new String[data_length][activity_length];
 	String[][] data_m3	= new String[data_length][activity_length];
 	String[][] data_m	= new String[data_length][activity_length];
-	
+
+                            
 //construction function
 	@SuppressWarnings("unchecked")
 	public Gui() throws BiffException, IOException{
+            
+            for(i=0;i<data_length;i++) {
+                for(int j=0;j<activity_length;j++){
+                    data_m1[i][j]="0";
+                    data_m2[i][j]="0";
+                    data_m3[i][j]="0";
+                }
+            }
```

Comment:
Declare the String value in data_m1, data_m2 and data_m3. All the default values are set to be "0".